### ISSUE TITLE
enable cos and sin on `no_std` builds

### ISSUE THREAD

**Issue by wucke13** [iliekturtles/uom#283]
Currently `sin` and `cos` functions are disabled on `no_std`. These are common functions which are easily available through `libm`, so it makes sense to expose them.

**Comment by iliekturtles** [comment]
They are currently disabled because the underlying implementations aren't available in [core::f32](https://doc.rust-lang.org/core/primitive.f32.html) like they are in [std::f32](https://doc.rust-lang.org/std/primitive.f32.html#method.sin). Is there some way to use `libm` easily for `no_std`?

**Comment by adamreichold** [comment]
I think the [`libm` crate](https://crates.io/crates/libm) can be used in `no_std` environments as it is a pure Rust reimplementation of MUSL's libm.

**Comment by wucke13** [comment]
Yes, further on there is a `libm` feature in `num`,  which can be easily used since `num` is a dependency already

**Comment by wucke13** [comment]
@iliekturtles Any chance that you take you _fix_ this?
